spec/filters/current_spec.rb

```diff
@@ -3,6 +3,11 @@
 describe "current" do
   before { visit "/current.html" }
 
+  it "selects a link without [aria-current]" do
+    expect(page).to have_selector :link, "Home", current: nil
+    expect(page).to have_no_selector :link, "About us", current: nil
```

Comment:
This may flag rubocop dual expectation failures. Probs better to split it into 2 specs. Especially as the it label is kind of wrong for this one

Reply:
@luke-hill thank you for the review! I've pushed up a change.